uber/templates/base.html

```diff
@@ -55,7 +55,6 @@
                     {Groups: '../groups/'},
                     {% if c.HAS_PEOPLE_ACCESS %}
                         {'All Untaken Shifts': '../jobs/everywhere'},
-                        {Staffers: '../registration/staffers'},
```

Comment:
This change was already proposed and a separate PR made in the RAMS repo, and I agree with the motivation behind it so I'm including it here.